README.md

```diff
@@ -20,7 +20,7 @@ It consists of two packages: a plain TypeScript core library ([phoenix-event-dis
 
 Phoenix is supported by the [HEP Software Foundation](https://hepsoftwarefoundation.org) and is the official web event display of the [ATLAS experiment](https://atlas.cern).
 
-It was selected for Google Summer of Code support in 2019 and 2020.
+It was selected for Google Summer of Code support in 2019, 2020 and 2021.
```

Comment:
Not sure if we should include 2021. There wasn't any student selected so no work was done. Maybe changing the phrasing to "It received support from Google Summer of Code in 2019 and 2020." The "selected" changes the meaning.

Reply:
shall I write it as "It received support from Google Summer of Code in 2019, 2020 and 2021." ?

Reply:
I actually think it's okay to say it was selected in 2019, 2020, and 2021... Phoenix was, but sadly we didn't match to a student in 2021. If we want to be completely honest, we could say "... selected for Google Summer of Code support in 2019, 2020 and 2021, and had students in 2019 and 2020." But I think the main point, from a PR point of view, was that Phoenix was considered worthy.